cmd/edit/cluster/cmd.go

```diff
@@ -463,6 +545,98 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	/*******  Registries Config *******/
+	updateRegistriesConfig := insecureRegistries != nil ||
+		blockedRegistries != nil ||
+		allowedRegistriesForImport != "" ||
+		allowedRegistries != nil ||
+		platformAllowlist != "" ||
+		additionalTrustedCa != ""
+
+	if !updateRegistriesConfig && interactive.Enabled() {
+		updateRegistriesConfigValue, err := interactive.GetBool(interactive.Input{
+			Question: "Update registries config",
+			Default:  updateRegistriesConfig,
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid value: %s", err)
+			os.Exit(1)
+		}
+		updateRegistriesConfig = updateRegistriesConfigValue
+	}
+
+	if updateRegistriesConfig && interactive.Enabled() {
+		allowedRegistriesInputs, err := interactive.GetString(interactive.Input{
+			Question: "Allowed Registries",
+			Help:     cmd.Flags().Lookup("allowed-registries").Usage,
+			Default:  allowedRegistries,
+		})
+		if err != nil {
+			r.Reporter.Errorf("expected a valid value for allowed registries: %s", err)
+			os.Exit(1)
+		}
+		allowedRegistries = helper.HandleEmptyStringOnSlice(strings.Split(allowedRegistriesInputs, ","))
+
+		blockedRegistriesInputs, err := interactive.GetString(interactive.Input{
+			Question: "Blocked Registries",
+			Help:     cmd.Flags().Lookup("blocked-registries").Usage,
+			Default:  blockedRegistries,
+		})
+		if err != nil {
+			r.Reporter.Errorf("expected a valid value for blocked registries: %s", err)
+			os.Exit(1)
+		}
+		blockedRegistries = helper.HandleEmptyStringOnSlice(strings.Split(blockedRegistriesInputs, ","))
+
+		insecureRegistriesInputs, err := interactive.GetString(interactive.Input{
+			Question: "Insecure Registries",
+			Help:     cmd.Flags().Lookup("insecure-registries").Usage,
+			Default:  insecureRegistries,
+		})
+		if err != nil {
+			r.Reporter.Errorf("expected a valid value for insecure registries: %s", err)
+			os.Exit(1)
+		}
+		insecureRegistries = helper.HandleEmptyStringOnSlice(strings.Split(insecureRegistriesInputs, ","))
+
+		allowedRegistriesForImport, err = interactive.GetString(interactive.Input{
+			Question: "Allowed Registries For Import",
+			Help:     cmd.Flags().Lookup("allowed-registries-for-import").Usage,
+			Default:  allowedRegistriesForImport,
+			Validators: []interactive.Validator{
+				ocm.ValidateAllowedRegistriesForImport,
+			},
+		})
+		if err != nil {
+			r.Reporter.Errorf("expected a valid value for allowed registries for import: %s", err)
+			os.Exit(1)
```

Comment:
Upon reviewing the reporter code, I noticed that the Reporter object outputs the error to the console. In my opinion, r.Reporter.Errorf should follow the convention of using an uppercase letter, though I'll defer that decision to someone with more familiarity with the project.

The Go best practice regarding lowercase error messages applies specifically to returned errors from functions, allowing them to be easily concatenated with other strings, as you're doing with the err variable in this case.

Reply:
Thanks @ziccardi 👍 Will review the output. If it's directly to console, will start with upper case